source/_components/keba.markdown

```diff
@@ -0,0 +1,97 @@
+---
+layout: page
+title: "Keba Charging Station"
+description: "Instructions on how to setup your Keba charging station with Home Assistant."
+date: 2019-06-11 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: keba.png
+ha_category:
+  - Binary Sensor
+  - Lock
+  - Sensor
+ha_release: 0.94
+redirect_from:
```

Comment:
Please remove the `redirect_from`

Reply:
Fixed, I probably used an old style documentation as template.

Reply:
Well what I mainly see is that people unquestionably take over without really knowing what it is for 😉